src/activities/binary_bulb/binary_bulb.js

```diff
@@ -0,0 +1,72 @@
+/* GCompris - binary_bulb.js
+ *
+ * Copyright (C) 2018 Rajat Asthana <rajatasthana4@gmail.com>
+ *
+ * Authors:
+ *   "RAJAT ASTHANA" <rajatasthana4@gmail.com> (Qt Quick port)
+ *
+ *   This program is free software; you can redistribute it and/or modify
+ *   it under the terms of the GNU General Public License as published by
+ *   the Free Software Foundation; either version 3 of the License, or
+ *   (at your option) any later version.
+ *
+ *   This program is distributed in the hope that it will be useful,
+ *   but WITHOUT ANY WARRANTY; without even the implied warranty of
+ *   MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ *   GNU General Public License for more details.
+ *
+ *   You should have received a copy of the GNU General Public License
+ *   along with this program; if not, see <http://www.gnu.org/licenses/>.
+ */
+.pragma library
+.import QtQuick 2.6 as Quick
+
+var toBeConverted = [1, 2, 3, 1, 4, 9, 13, 15, 57, 152, 248, 239]
```

Comment:
that should be separated by level, not having all the values in a 1 dimension array